client/my-sites/upgrades/checkout-thank-you/footer.jsx

```diff
@@ -8,57 +8,19 @@ import React from 'react';
  */
```

Comment:
Now that we have `components/happiness-support`, what are the benefits of keeping this component, instead of mounting `HappinessSupport` directly in `CheckoutThankYou`? I believe all it would take is replacing

```js
<CheckoutThankYouFooter purchases={ purchases } />
```

with

```js
<Card className="checkout-thank-you-footer">
  <HappinessSupport isJetpack={ purchases && purchases.some( isJetpack ) } />
</Card>
```

This does increase the size of `CheckoutThankYou` (by two lines), but gets rid of a component that is only mounted here, and makes it easier to understand the render() by removing an unnecessary level. I might be missing some of the benefits of the current implementation, though.